pref_sources/RubyMine/codestyles/Pivotal.xml

```diff
@@ -72,6 +67,7 @@
   <codeStyleSettings language="ruby">
     <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
     <option name="SPACE_WITHIN_BRACES" value="true" />
+    <option name="SPACE_WITHIN_ARRAY_INITIALIZER_BRACES" value="true" />
```

Comment:
What's an example of the code that this will produce?

Reply:
*Before*
```
arr.each do {|el| puts el}
```

*After*
```
arr.each do { |el| puts el }
```

The after is what our styleguide says to do https://github.com/mavenlink/welcome/blob/5ea8268a548db75694375f9c4b354d8cec4c31d9/code/style/ruby.md#style

Reply:
Yeah, that makes sense. The word "array" in there was confusing me, though. `array_initializer_braces`?

Reply:
Yeah that is weird. I'll double check on this.

Reply:
@ahuth Yup, oddly enough, when checking/unchecking the `Spaces around curly braces in blocks` option, that's the config setting that gets changed. Maybe a form of legacy code in RubyMine or IntelliJ-based IDEs. There doesn't appear to be an option to do what that actually describes.

Reply:
Okay, thanks for looking into it

Reply:
Weeeeird.